### DISCUSSION TITLE
Can this run on the iGPU?

### DISCUSSION THREAD

**Discussion by markgabrang** [VirtualDrivers/Virtual-Display-Driver#122] [Q&A] [answered]
Hi @itsmikethetech , thank you for your work!

I was wondering if this can run on a separate GPU (iGPU perhaps) to eliminate the overhead on the main GPU?

Cheers.

**Comment by markgabrang** [comment]
Oh I missed this part, sorry!:

[Windows 11 23H3+ (HDR)](https://github.com/itsmikethetech/Virtual-Display-Driver/blob/master) - Coming Soon, adds support for manual GPU switching.

**Comment by bud3699** [comment, marked as answer]
Yup, got added a while back
